options.go

```diff
@@ -130,7 +130,7 @@ func getDefaultOptions(src, dest string) Options {
 		OnError:           nil,                // Default is "accept error"
 		Skip:              nil,                // Do not skip anything
 		AddPermission:     0,                  // Add nothing
-		PermissionControl: PerservePermission, // Just preserve permission
+		PermissionControl: PreservePermission, // Just preserve permission
```

Comment:
In order to keep backward compatibility, we can still keep `PerservePermission` with `// Deprecated:` marker.

Reply:
Accordingly, let's painfully respect `PerservePermission` but prioritize `PreservePermission` in other code blocks.